corehq/apps/reports/standard/export.py

```diff
@@ -474,3 +527,45 @@ def selected_exports_data(self):
                 'exporttype': 'form',
             } for export in self.exports
         }
+
+
+class NewCaseExportReport(CaseExportReport):
```

Comment:
This is going to fully replace `CaseExportReport` at some point (soon), right? Otherwise, a more descriptive name would help out.